plugins/modules/aci_access_spine_switch_selector.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Eric Girard <@netgirard>
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "certified",
+}
+
+DOCUMENTATION = r"""
+---
+module: aci_access_spine_switch_selector
+short_description: Manage Fabric Access Policy Spine Switch Port Selectors (infra:SpineS and infra:RsSpineAccNodePGrp)
```

Comment:
same as other comment

Reply:
Will remove it as explained in the previous comment.